build.gradle

```diff
@@ -27,4 +30,5 @@ dependencies {
     compile 'joda-time:joda-time:2.9.4'
 
     testCompile 'org.testng:testng:6.8.21'
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlin_version"
```

Comment:
Does it mean that the library won't be usable with java 6 & 7 projects? I'd prefer to keep the compatibility.